frontend/packages/console-metal3/src/components/host.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import {
+  BaremetalHostRole,
+  BaremetalHostStatus,
+  getHostMachineName,
+  getHostBmcAddress,
+  getName,
+  getNamespace,
+  getResource,
+  getSimpleHostStatus,
+  getUid
+} from 'kubevirt-web-ui-components';
```

Comment:
We need to add this dependency into the appropriate `package.json` file.

For now, all **core** Console dependencies are declared in the monorepo root package. These dependencies reflect the requirements of application code within the `public` directory, to be moved into the `@console/app` package in future.

As for `kubevirt-web-ui-components`, this is a plugin-specific dependency so please add it to Metal3 plugin's `package.json` file.

Reply:
Done